lua/starfall/libs_sh/angles.lua

```diff
@@ -46,7 +46,10 @@ end
 -- String based indexing returns string, just a pass through.
 local pyr = { p = 1, y = 2, r = 3, pitch = 1, yaw = 2, roll = 3 }
 
---- __newindex metamethod
+--- Sets a value at a key in the angle
```

Comment:
Do the metamethods appear in the docs? I don't remember if that was implemented or not.

Reply:
Probably need `-- @name ang_methods.__newindex`

Reply:
Nvm, looks like we have `@libtbl ang_meta` for that.